axelor-human-resource/src/main/java/com/axelor/apps/hr/service/project/ProjectPlanningTimeServiceImpl.java

```diff
@@ -655,4 +655,36 @@ protected boolean isValidProjectTask(ProjectTask projectTask) {
         && projectTask.getAssignedTo() != null
         && projectTask.getAssignedTo().getEmployee() != null;
   }
+
+  @Override
+  public BigDecimal getTotalPlannedTimeInDaysForPeriod(
+      List<ProjectTask> projectTaskList, LocalDate fromDate, LocalDate toDate)
+      throws AxelorException {
+
+    BigDecimal totalPlannedTime = BigDecimal.ZERO;
+
+    List<ProjectPlanningTime> projectPlanningTimeList =
+        planningTimeRepo.findByProjectTasksAndPeriod(projectTaskList, fromDate, toDate).fetch();
+
+    if (CollectionUtils.isNotEmpty(projectPlanningTimeList)) {
+      Unit dayUnit = appBaseService.getAppBase().getUnitDays();
+
+      for (ProjectPlanningTime projectPlanningTime : projectPlanningTimeList) {
+        totalPlannedTime =
+            totalPlannedTime.add(
+                getPlannedTimeInTargetUnit(
+                    projectPlanningTime.getTimeUnit(),
+                    dayUnit,
+                    projectPlanningTime.getPlannedTime()));
+      }
+    }
+
+    return totalPlannedTime;
+  }
+
+  protected BigDecimal getPlannedTimeInTargetUnit(Unit startUnit, Unit endUnit, BigDecimal value)
+      throws AxelorException {
+
+    return unitConversionForProjectService.convert(startUnit, endUnit, value, value.scale(), null);
```

Comment:
Need the project here, else you will get exception if time unit conversion ask for project.numberHoursADay